specs/src/specs/state_machine_modules.md

```diff
@@ -0,0 +1,30 @@
+# State Machine Modules
+
+Celestia app is built using the cosmos-sdk, and follows standard cosmos-sdk module structure.
+
+## `celestia-app` Specific Modules
+
+- [blob](../../../x/blob/README.md)
+- [qgb](../../../x/qgb/README.md)
+- [mint](../../../x/mint/README.md)
+- [paramfilter](../../../x/paramfilter/README.md)
+- [upgrade](../../../x/upgrade/README.md)
+
+## Standard `cosmos-sdk` Modules
+
+- [auth](https://github.com/celestiaorg/cosmos-sdk/blob/v1.14.0-sdk-v0.46.11/x/auth/spec/README.md)
+- [authz](https://github.com/celestiaorg/cosmos-sdk/blob/v1.14.0-sdk-v0.46.11/x/authz/spec/README.md)
+- [bank](https://github.com/celestiaorg/cosmos-sdk/blob/v1.14.0-sdk-v0.46.11/x/bank/spec/README.md)
+- [capability](https://github.com/celestiaorg/cosmos-sdk/blob/v1.14.0-sdk-v0.46.11/x/capability/spec/README.md)
+- [crisis](https://github.com/celestiaorg/cosmos-sdk/blob/v1.14.0-sdk-v0.46.11/x/crisis/spec/README.md)
+- [distribution](https://github.com/celestiaorg/cosmos-sdk/blob/v1.14.0-sdk-v0.46.11/x/distribution/spec/README.md)
+- [evidence](https://github.com/celestiaorg/cosmos-sdk/blob/v1.14.0-sdk-v0.46.11/x/evidence/spec/README.md)
+- [feegrant](https://github.com/celestiaorg/cosmos-sdk/blob/v1.14.0-sdk-v0.46.11/x/feegrant/spec/README.md)
+- [genutil](https://github.com/celestiaorg/cosmos-sdk/tree/v1.14.0-sdk-v0.46.11/x/genutil) (no spec)
```

Comment:
Is this really a module?

Reply:
I was tempted to not include it, but yeah it is and it can result in non-determinism for starting the genesis so I decided to include it.

https://github.com/celestiaorg/cosmos-sdk/blob/24903f4a983911f708fbf138a895ea4abc5dc752/x/genutil/module.go#L45-L53